spec/lib/task_helpers/imports/custom_buttons_spec.rb

```diff
@@ -19,19 +24,22 @@
       end
 
       context "with existing identical buttons" do
-        it 'should raise' do
+        it 'should NOT raise' do
           TaskHelpers::Imports::CustomButtons.new.import(options)
-          assert_raises_import_error
+          assert_imports_only_custom_button_set_one
         end
       end
 
       context "yaml import failure" do
         it 'should raise' do
-          file = Tempfile.new('foo.yaml', data_dir)
-          file.write("bad yaml here")
-          TaskHelpers::Imports::CustomButtons.new.import(options)
-          assert_raises_import_error
-          assert_imports_only_custom_button_set_one
+          file = Tempfile.new(['foo', '.yaml'], data_dir)
```

Comment:
@borisko123 Can you use `Tempfile` with the `create` method?  It would handle the cleanup work.

I didn't test this, but I think you can replace it with:
```ruby
Tempfile.create(['foo', '.yaml'], data_dir) do |file|
  file.write("bad yaml here")
  assert_raises_import_error
end

```

Reply:
Nice. Will try and fix

Reply:
done